index.html

```diff
@@ -0,0 +1,279 @@
+                </ul>
+            </div>
+        </div>
+        <div class="typo-container" id="typos-id">
+            <h1>Typography</h1>
+            <div class='headings'>
+                <h1>Heading 1</h1>
+                <h2>Heading 2</h2>
+                <h3>Heading 3</h3>
+                <h4>Heading 4</h4>
+                <h5>Heading 5</h5>
+                <h6>Heading 6</h6>
+            </div>
+            <div class="text-vary">This is <small>small </small><strong>important </strong>
+                <mark>but</mark><b> bold </b> <i>italic </i> text</div>
+            <div class="text-vary text-color">The text colour is grey</div>
+        </div>
+        <div class="grid-container" id="grids-id">
+            <h1>Grids</h1>
+            <h2>Grid with 2 element</h2>
+            <div class="grid-2">
+                <div class="grid-area grid-2-area-1">Area 1</div>
+                <div class="grid-area grid-2-area-2">Area 2</div>
+            </div>
+            <h2>Grid with 3 element</h2>
+            <div class="grid-3">
+                <div class="grid-area grid-3-area-1">Area 1</div>
+                <div class="grid-area grid-3-area-2">Area 2</div>
+                <div class="grid-area grid-3-area-3">Area 3</div>
+            </div>
+        </div>
+        <div class="image-container" id='image-id'>
+            <h1>Images</h1>
+            <h2>Responsive Image</h2>
+            <img src="/assets/northernlights.jpg" class="img-responsive">
+            <h2>Round Image</h2>
+            <img src="/assets/northernlights.jpg" class="img-round">
+        </div>
+        <div class="badges-container" id='badges-id'>
+            <h1>Badges</h1>
+            <h2>Badges on icons</h2>
+            <div class="badge-icon">
+                <img src="/assets/message.svg">
+                <span class="badge-icon-count">99</span>
+            </div>
+            <h2>Badges on Avatars</h2>
+            <div class="badge-div">
+                <img src="/assets/rdj.jpg" class="badge-img badge-img-lg" />
+                <span class="badge-image left">Handsome</span>
+            </div>
+        </div>
+        <div class="modals-container" id='modals-id'>
+            <h1>Modals</h1>
+            <button id="myBtn" onclick="buttonOnClick('myBtn')">Open Modal</button>
+            <div id="myModal" class="modal">
+                <div class="modal-content">
+                    <span class="close" onclick="spanOnClick('close')">&times;</span>
+                    <p>Some text in the Modal..</p>
+                </div>
+            </div>
+        </div>
+        <div class ='button-container' id='buttons-id'>
+            <h1>Buttons</h1>
+            <button class="primary-btn">Primary </button>
+            <br></br>
+            <a href="#header" class='link-button'>Link Button</a>
+            <br></br>
+            <button class="icon-btn"><i class="fa fa-bars"></i> Menu</button>
```

Comment:
Check out examples of hamburger buttons and try to style it similar to them. Up to you though, this works as well...